brycemara/refactor-tractor
# Add refactor of scripts.js

## What is the change?
refactor of scripts.js for readability and SRP management 

## What does it fix?
1. duplicate methods 
1. readability 
1. naming conventions
## Is this a fix or a feature? 
refactor / fix
## Where should the reviewer start?
* scripts.js line 19
## How should this be tested?
* in terminal run`npm run build` 
* then `npm start` to view the page on a local host 
* open the console and check for errors
* make sure all visuals load and populate
@brycemara @dietza @coopterrones
